### ISSUE TITLE
Mentions.getMentions only returns '@' when multiple prefixes are used

### ISSUE THREAD

**Issue by N00nDay** [ant-design/ant-design#26473]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.
-- There is another issue posted for a previous version of antd but was closed stating the newer version of antd resolved the issue. 

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/fast-water-v5cnh)

### Steps to reproduce
Type "#something" in the input
Type "@something" in the input
Use "getMentions" method to get mentions array list
It only contains the one (or the ones) with '@'

### What is expected?
I expect to receive an array containing 2 objects, one for prefix "@" and another for prefix "#"

### What is actually happening?
An array with only 1 object is being returned containing the prefix "@"

| Environment | Info |
|---|---|
| antd | 4.6.1 |
| React | 16.12.0 |
| System | Mac Catalina 10.15.5 |
| Browser | Chrome 83.0.4103.116 |

**Comment by N00nDay** [comment]
It may be a lack of documentation or an issue not sure at this point. After looking at the code I have found that if I pass an object with the prefix list that this works as I would expect it to.

Instead of:
`Mentions.getMentions(value)`

I used:
`Mentions.getMentions(value, { prefix: ["@", "#"]})`

Is this the correct way to use this or is `getMentions` supposed to grab the prefixes from the props of the `Mention` tag itself?

The only example of `getMentions` in the documentation is using 1 prefix which means the second argument is not necessary.
